src/pages/logout/logout.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {
+    useHistory,
+} from 'react-router-dom';
+
+function Logout() {
+    const history = useHistory();
+    function HandleClick() {
```

Comment:
de HandleClick function moet niet met een hoofdletter, en in deze functie moet de toggleIsAuthenticated naar false gezet worden door dit binnen de acolades van de functie te zetten:

toggleIsAuthenticated(false)